frontend/src/app/questions/questions.component.ts

```diff
@@ -111,6 +111,7 @@ export class QuestionsComponent implements OnInit {
         if (
             !this.question.title?.trim() ||
             !this.question.topics ||
+            this.question.topics?.length == 0 ||
             !this.question.difficulty?.trim() ||
             !this.question.description?.trim()
         ) {
```

Comment:
I think this could piggyback off the form validators too and do a `if(!questionFormGroup.valid)`. Perhaps we just trim the fields directly then check for validity so that the submitted data is trimmed too.

Reply:
Didn't realise formGroup could be used this way 😅. Thanks for pointing out.

Reply:
yeah Bryan and I discovered this while working on login and registration, pretty cool!